python_modules/libraries/dagster-dbt/dagster_dbt/include/dbt_scaffold/definitions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from dagster import Definitions
+
+from dagster_dbt import DbtCli, DbtManifest, dbt_assets
+
+DBT_PROJECT_DIR = Path(__file__).parent.joinpath("..", "jaffle_shop")
+DBT_MANIFEST_PATH = DBT_PROJECT_DIR.joinpath("target", "manifest.json")
+DBT_MANIFEST = DbtManifest.read(path=DBT_MANIFEST_PATH)
+
+
+@dbt_assets(manifest=DBT_MANIFEST)
+def my_dbt_assets(dbt: DbtCli):
+    yield from dbt.cli(["build"], manifest=DBT_MANIFEST).stream()
+
+
+schedules = [
+    DBT_MANIFEST.build_schedule(
+        job_name="materialize_dbt_models",
+        cron_schedule="0 0 0 * *",
+    )
+]
+
+defs = Definitions(
+    assets=[my_dbt_assets],
+    schedules=schedules,
+    resources={
+        "dbt": DbtCli(
+            project_dir=DBT_PROJECT_DIR.as_posix(),
```

Comment:
Is there some way we can avoid the need for `as_posix` here?

Reply:
Because we are using `pathlib`, this is not a primitive type that can be expressed in our Dagster config system. So currently, this is blocked by https://github.com/dagster-io/dagster/issues/14564.

If we want to continue using this as a string, then we need to use `file_relative_path`. I avoided this since I want to deprecate this function (https://github.com/dagster-io/dagster/pull/14499), but we can use this if we want.

Reply:
Got it - ok not the end of the world for now to use `as_posix`